### PR TITLE
Got modal working! Still needs much styling...

### PR DESCRIPTION
**Pull request by macrawford**

### CODE REVIEW

File: client/src/components/Answers.jsx [hunk 1]
```diff
@@ -91,6 +92,49 @@ class Answers extends React.Component {
       showAll: true
     })
   }
+  convertToMonthYear(date) {
+    var split = date.split('-')
```
Comment (jnweiner) [inline client/src/components/Answers.jsx:96]: I bet you could use a switch statement here!

Comment (jnweiner) [reply client/src/components/Answers.jsx:96]: or, you could strip the 0 off the front of the split, put the month names into an array (starting with a dummy value in the zero index), and then you could refer to each month by monthsArray[split] ... just an idea if you wanted to shorten this! (ofc the way it currently is works great too!)

Comment (macrawford) [reply client/src/components/Answers.jsx:96]: I should look into that...

File: client/src/components/AskQuestion.jsx [hunk 2]
```diff
@@ -14,17 +13,69 @@ const TextField = styled.textarea `
...
 
 const SubmitQButton = styled.button`
+  background-color: rgb(0, 0, 0);
+  border-color: rgb(0, 0, 0);
+  border-radius: 3px;
+  border-width: 1px;
+  display: block;
+  color: rgb(255, 255, 255);
+  // Will need to fix float
+  padding-bottom: 8px;
```
Comment (jnweiner) [inline client/src/components/AskQuestion.jsx:23]: just heads up you can do all of padding for all 4 sides in just one css property! this works for margin and border (and i think a few other things too) https://developer.mozilla.org/en-US/docs/Web/CSS/padding

File: client/src/components/AskQuestion.jsx [hunk 3]
```diff
@@ -33,28 +84,79 @@ class AskQuestion extends React.Component {
...
-    this.props.addQuestion(this.state.question);
+    this.props.addQuestion(question);
     this.reinitializeState();
   }
   reinitializeState() {
     this.setState({
       question: ''
     })
   }
+  showModal() {
```
Comment (jnweiner) [inline client/src/components/AskQuestion.jsx:97]: if you wanted, you could make this into just one toggleModal function that just changes showModal to be !this.state.showModal (the opposite)

**Comment by jnweiner** [comment]
just made a few comments if you wanted to shorten a few areas. looks great!

File: client/src/components/AskQuestion.jsx (hunk 2 above)
Comment (macrawford) [reply client/src/components/AskQuestion.jsx:23]: Yeah lol I know I've been frantically doing css I should clean it up next week

File: client/src/components/AskQuestion.jsx (hunk 3 above)
Comment (macrawford) [reply client/src/components/AskQuestion.jsx:97]: Good call

**Pull request merged**